src/test/java/io/beanmapper/BeanMapperTest.java

```diff
@@ -1619,6 +1619,17 @@ public void emptyListToExistingList() {
         assertEquals(targetItems, target.getItems());
     }
 
+    @Test
+    public void nullListToNullList() {
+        BeanMapper beanMapper = new BeanMapperBuilder()
+                .setUseCollectionNullValue(false)
```

Comment:
als `useCollectionNullValue` true is moet volgens mij de value null zijn. Dus als het false is juist niet. 
Volgens mij werkt het nu precies verkeerd om...

Reply:
Default gebruikt ```useNullValue``` true en worden null waarden daarom niet genegeerd. Ik neem aan dat dit op dezelfde manier werkt, klopt dat of denk ik verkeerd?